### ISSUE TITLE
When delete button on call history is pressed, the effect will be shown only after closing and reopening Phone

### ISSUE THREAD

**Issue by e075e** [FossifyOrg/Phone#158]
### Checklist

- [X] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Phone/releases/latest).
- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Phone/issues) or [closed](https://github.com/FossifyOrg/Phone/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Phone/discussions) or [closed](https://github.com/FossifyOrg/Phone/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Phone/blob/master/CONTRIBUTING.md).

### Affected app version

1.1.1

### Affected Android/Custom ROM version

Android 13 / eOS 1.20

### Affected device model

samsung a52q

### How did you install the app?

F-Droid / IzzyOnDroid

### Steps to reproduce the bug

1. open call history
2. make a call if you have none
3. press the delete button on top right corner
4. confirm on the popup

### Expected behavior

call history will be deleted immediately, showing nothing on the call history

### Actual behavior

call history is deleted, but to show the effect you have to close Phone from task manager and reopening it

### Additional information

As a workaround, you can select the entry and delete it.

Could this issue be related with #126 ?

**Comment by naveensingh** [comment]
Yes, it's related to the changes in #126. It'll be fixed in the next release.

Closing as a duplicate of https://github.com/FossifyOrg/Phone/issues/146
